src/typings/stamp__it/index.d.ts

```diff
@@ -0,0 +1,575 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/method-signature-style */
+/* eslint-disable @typescript-eslint/no-invalid-void-type */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-use-before-define */
+/* eslint-disable max-len */
+
+declare module '@stamp/it' {
+  // Type definitions for stampit 4.3
+  // Project: https://github.com/stampit-org/stampit, https://stampit.js.org
+  // Definitions by: Vasyl Boroviak <https://github.com/koresar>
+  //                 Harris Lummis <https://github.com/lummish>
+  //                 PopGoesTheWza <https://github.com/popgoesthewza>
+  // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+  // TypeScript Version: 3.5
+
+  // Utility types
+
+  /**
+   * @internal Base type for all `methods`-like metadata.
+   * @template This The type to use for `this` within methods.
+   */
+  interface MethodMap<This> {
+    [s: string]: ((this: This, ...args: any[]) => any) | {}
+  }
+
+  /** @internal A plain old JavaScript object created by a `Stamp`. */
+  type Pojo = object // { [s: string]: any; }
+
+  /** @internal Base type for all `properties`-like metadata. */
+  // TODO: discriminate Array
+  type PropertyMap = object // { [s: string]: any; }
+
+  /** @internal Signature common to every `Stamp`s. */
+  interface StampSignature {
+    (options?: PropertyMap, ...args: unknown[]): any
+    compose: ComposeMethod & stampit.Descriptor<any, any>
+  }
+
+  /**
+  * @template Original The type to extract the `Stamp` type from.
+  */
+  type StampType<Original> = Original extends /* disallowed types */[] | bigint
+    ? never
+    : stampit.IsADescriptor<Original> extends true
+      ? (Original extends stampit.ExtendedDescriptor<infer Obj, infer Stamp> ? Stamp : never)
+      : unknown extends Original /* is any or unknown */
+        ? stampit.Stamp<Original>
+        : Original extends StampSignature
+          ? Original
+          : Original extends stampit.ExtendedDescriptor<infer Obj, any>
+            ? stampit.Stamp<Obj>
+            : Original extends Pojo
+              ? stampit.Stamp<Original> /* assume it is the object from a stamp object */
+              : never
+
+  /**
+  * A factory function to create plain object instances.
+  * @template Obj The object type that the `Stamp` will create.
+  */
+  type FactoryFunction<Obj> = (options?: PropertyMap, ...args: any[]) => stampit.StampObjectType<Obj>
+
+  /**
+  * @internal Chainables `Stamp` additionnal methods
+  * @template Obj The object type that the `Stamp` will create.
+  */
+  type StampChainables<Obj> = Chainables<stampit.StampObjectType<Obj>, StampType<Obj>>
+
+  /**
+  * @internal Chainables `Stamp` additionnal methods
+  * @template Obj The object type that the `Stamp` will create.
+  * @template S̤t̤a̤m̤p̤ The type of the `Stamp` (when extending a `Stamp`.)
+  */
+  interface Chainables<Obj, S̤t̤a̤m̤p̤ extends StampSignature> {
```

Comment:
Why "S̤t̤a̤m̤p̤" is not ASCII?

Reply:
Interesting, the thing is this was autogenerated. I guess I will retype those occurences to fix the issue